src/components/cc-logs-instances/cc-logs-instances.stories.js

```diff
@@ -0,0 +1,328 @@
+};
+
+const DEPLOYING = [
+  createInstance(DEPLOYMENTS.wip, 0, 'BUILD', 'STOPPING', 'Rich medicham'),
+  createInstance(DEPLOYMENTS.wip, 0, 'RUN', 'UP', 'Easy hydreigon'),
+  createInstance(DEPLOYMENTS.wip, 1, 'RUN', 'READY', 'Long delphox'),
+  createInstance(DEPLOYMENTS.wip, 2, 'RUN', 'DEPLOYING', 'Curly honchkrow'),
+];
+
+const RUNNING = [
+  createInstance(DEPLOYMENTS.ok1, 0, 'RUN', 'UP', 'Easy hydreigon'),
+  createInstance(DEPLOYMENTS.ok1, 1, 'RUN', 'UP', 'Long delphox'),
+  createInstance(DEPLOYMENTS.ok1, 2, 'RUN', 'UP', 'Curly honchkrow'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'BUILD', 'DELETED', 'Rich medicham'),
+];
+
+const SCALE_DOWN = [
+  createInstance(DEPLOYMENTS.ok1, 0, 'RUN', 'UP', 'Easy hydreigon'),
+  createInstance(DEPLOYMENTS.ok1, 1, 'RUN', 'UP', 'Long delphox'),
+  createInstance(DEPLOYMENTS.ok1, 2, 'RUN', 'STOPPING', 'Curly honchkrow'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'BUILD', 'DELETED', 'Rich medicham'),
+];
+
+const SCALE_UP = [
+  createInstance(DEPLOYMENTS.wip, 2, 'RUN', 'DEPLOYING', 'Rapid combee'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'RUN', 'UP', 'Easy hydreigon'),
+  createInstance(DEPLOYMENTS.ok1, 1, 'RUN', 'UP', 'Long delphox'),
+  createInstance(DEPLOYMENTS.ok1, 2, 'RUN', 'DELETED', 'Curly honchkrow'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'BUILD', 'DELETED', 'Rich medicham'),
+];
+
+const APPLICATION_STOPPED = [
+  createInstance(DEPLOYMENTS.ok1, 0, 'RUN', 'DELETED', 'Easy hydreigon'),
+  createInstance(DEPLOYMENTS.ok1, 1, 'RUN', 'DELETED', 'Long delphox'),
+  createInstance(DEPLOYMENTS.ok1, 2, 'RUN', 'DELETED', 'Curly honchkrow'),
+  createInstance(DEPLOYMENTS.ok2, 2, 'RUN', 'DELETED', 'Rapid combee'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'BUILD', 'DELETED', 'Rich medicham'),
+];
+
+const FAILED_DEPLOYMENT = [
+  createInstance(DEPLOYMENTS.failed, 0, 'RUN', 'DELETED', 'Petite misdreavus'),
+  createInstance(DEPLOYMENTS.failed, 1, 'RUN', 'DELETED', 'Happy meowstic-female'),
+  createInstance(DEPLOYMENTS.failed, 2, 'RUN', 'DELETED', 'Rapid combee'),
+];
+
+const CANCELLED_DEPLOYMENT = [
+  createInstance(DEPLOYMENTS.cancelled, 0, 'RUN', 'DELETED', 'Crispy slowking'),
+  createInstance(DEPLOYMENTS.cancelled, 1, 'RUN', 'DELETED', 'Glassy delcatty'),
+  createInstance(DEPLOYMENTS.cancelled, 2, 'RUN', 'DELETED', 'Excited oshawott'),
+];
+
+const ALL_INSTANCES = [
+  createInstance(DEPLOYMENTS.wip, 0, 'BUILD', 'STOPPING', 'Proud electrode'),
+  createInstance(DEPLOYMENTS.wip, 0, 'RUN', 'UP', 'Difficult claydol'),
+  createInstance(DEPLOYMENTS.wip, 1, 'RUN', 'READY', 'Rich medicham'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'RUN', 'UP', 'Easy hydreigon'),
+  createInstance(DEPLOYMENTS.ok1, 1, 'RUN', 'UP', 'Long delphox'),
+  createInstance(DEPLOYMENTS.ok1, 2, 'RUN', 'STOPPING', 'Curly honchkrow'),
+  createInstance(DEPLOYMENTS.ok1, 0, 'BUILD', 'DELETED', 'Rich medicham'),
+  ...FAILED_DEPLOYMENT,
+  ...CANCELLED_DEPLOYMENT,
+];
+
+export const defaultStory = makeStory(conf, {
```

Comment:
**🎯 suggestion:** We may need  an instance in ghost mode, to test proper rendering.

Reply:
You're right, but after thinking about it a bit more, I think we should discuss it with others:
- In which section should ghost instances be placed? (running or deleted)
- Do we really want to show this information to the user?